### DISCUSSION TITLE
Reverse arrows no longer showing after 2021.927.0

### DISCUSSION THREAD

**Discussion by Jeglerjeg** [ppy/osu#14862] [Q&A]
My skin no longer shows any reverse arrows in gameplay.
Link to the skin: https://cloud.heckin.how/s/dNwbE22yb5PtKyC

**Comment by LittleEndu** [comment]
Your skin doesn't have an `reversearrow.png` element. For an quick fix, just add one from some other skin or copy from the [default legacy skin.](https://raw.githubusercontent.com/ppy/osu-resources/master/osu.Game.Resources/Skins/Legacy/reversearrow%402x.png)

The real fix involves making sure an `ISkin` that is an `OsuLegacySkinTransformer` does not fail to cast into `ISkinSource` so the fallback to legacy default reverse arrow works.
https://github.com/ppy/osu/blob/6f2d818b64705788414855ba726de63a9ebc60e5/osu.Game/Skinning/LegacySkinExtensions.cs#L33-L40

**Reply by bdach** [reply]
https://github.com/ppy/osu/pull/14865
